src/main/java/org/jenkinsci/plugins/authorizeproject/strategy/SpecificUsersAuthorizationStrategy.java

```diff
@@ -63,7 +63,14 @@
 public class SpecificUsersAuthorizationStrategy extends AuthorizeProjectStrategy {
     private static Logger LOGGER = Logger.getLogger(SpecificUsersAuthorizationStrategy.class.getName());
     private final String userid;
-    
+

```

Comment:
@casz I have two questions:

a) Assumed we don't find a better way to make this class JCasC-compatible:
Is this a future-proof "workaround" or can you think of possible changes in JCasC that would render this not working anymore?

b) If changing the type of `useApitoken` from `Boolean` to `boolean` this field is exported with value `0` by JCasC.
Is this a bug in JCasC or can you otherwise explain the details?

Reply:
@casz let me try to ping you.

Reply:
Or @timja can you provide some insight here?

Reply:
haven't reviewed yet but normally you need to use `boolean` for it to work with jcasc, I haven't looked into why though.

Reply:
I think you have a few options to prevent it being export,
You can annotate the getter with a `@Restrict(SomeRestriction.class)` https://github.com/jenkinsci/configuration-as-code-plugin/blob/master/plugin/src/main/java/io/jenkins/plugins/casc/BaseConfigurator.java#L324-L333
You can make the getter package-private
You can update the jcasc blacklist to hide it (shouldn't be needed)
I guess transient could work I've never tried it

Reply:
Transient should prevent it from being exported.
Another option is `@Restrict` as mentioned by @timja